### PR TITLE
dev-libs/pegtl: Bump to 3.2.0

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: dev-libs/pegtl/pegtl-3.2.0.ebuild [hunk 1]
```diff
@@ -13,16 +13,10 @@ LICENSE="MIT"
...
 KEYWORDS="~amd64 ~arm ~x86"
 IUSE="test"
-
 RESTRICT="!test? ( test )"
 
 S="${WORKDIR}/PEGTL-${PV}"
 
-PATCHES=(
-	"${FILESDIR}/${P}-cmake.patch"
-	"${FILESDIR}/${P}-gcc-10.patch" # bug 733678
```
Comment (jsmolic) [inline dev-libs/pegtl/pegtl-3.2.0.ebuild:23]: This is upstreamed

File: dev-libs/pegtl/pegtl-3.2.0.ebuild [hunk 2]
```diff
@@ -13,16 +13,10 @@ LICENSE="MIT"
...
-)
-
 src_configure() {
 	local mycmakeargs=(
 		-DPEGTL_INSTALL_CMAKE_DIR="$(get_libdir)/cmake/${PN}"
```
Comment (SoapGentoo) [inline dev-libs/pegtl/pegtl-3.2.0.ebuild:22]: if you're bored, try checkign whether upstream is willing to go for `GNUInstallDirs` (but I'm not asking you to do this)

**Review by SoapGentoo — APPROVED** [review]

File: dev-libs/pegtl/pegtl-3.2.0.ebuild [hunk 3]
```diff
@@ -13,16 +13,10 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~x86"
 IUSE="test"
-
 RESTRICT="!test? ( test )"
 
 S="${WORKDIR}/PEGTL-${PV}"
```
Comment (SoapGentoo) [inline dev-libs/pegtl/pegtl-3.2.0.ebuild:18]: try `S="${WORKDIR}/${P^^}"` instead

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-libs/pegtl/pegtl-3.2.0.ebuild (hunk 3 above)
Comment (jsmolic) [reply dev-libs/pegtl/pegtl-3.2.0.ebuild]: Thanks, added

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**